arc/src/CuriePME.c

```diff
@@ -0,0 +1,288 @@
+// Copyright (c) 2017, Intel Corporation.
+
+#include "CuriePME.h"
+
+static uint16_t nsr_save = 0;
+
+// Default initializer
+void CuriePME_begin(void)
```

Comment:
And again we're using capitals anywhere else in C function names... "curie_pme_". And I would leave off void from arg list. :)